app/services/login/after_login_actions.rb

```diff
@@ -31,10 +31,7 @@ def login_type
     end
 
     def evss_create_account
-      if current_user.authorize(:evss, :access?)
-        auth_headers = EVSS::AuthHeaders.new(current_user).to_h
-        EVSS::CreateUserAccountJob.perform_async(auth_headers)
-      end
+      EVSS::CreateUserAccountJob.perform_async(current_user.uuid) if current_user.authorize(:evss, :access?)
```

Comment:
Can you synchronized the [Sidekiq retry duration](https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry) with the [data's TTL](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/redis.yml#L6)?

Reply:
For example, if the data ttl is 24 hours, we shouldn't let the job retry past the 13th retry.
<img width="410" alt="ScreenShot-20221028T173607Z@2x" src="https://user-images.githubusercontent.com/28198/198698401-df8951ea-fd69-4c52-aa06-0440e093c032.png">

Reply:
Sounds good. I actually dramatically dropped the retries from that, since it doesn't make sense to be calling EVSS over and over when this fails

Reply:
I didn't even realize you did that! Haha.

Just this week, [we upped EVSS Doc Upload retries](https://github.com/department-of-veterans-affairs/vets-api/pull/10980) to at least 1 hour between attempts.